shared/localization/format.js

```diff
@@ -165,11 +165,15 @@ function _preformatValues(messageFormatter, values, lhlMessage) {
  * is assumed to already be in the given locale.
  * If you need to localize a messagem `getFormatted` is probably what you want.
  * @param {string} message
- * @param {Record<string, string | number>} values
+ * @param {Record<string, string | number>|undefined} values
  * @param {LH.Locale} locale
  * @return {string}
  */
-function formatMessage(message, values = {}, locale) {
+function formatMessage(message, values, locale) {
+  // Parsing and formatting can be slow. Don't attempt if the string can't
+  // contain ICU placeholders, in which case formatting is already complete.
+  if (!message.includes('{') && values === undefined) return message;
```

Comment:
check that `values` is undefined so it preserves the behavior of [throwing on invalid values for messages with no placeholders](https://github.com/GoogleChrome/lighthouse/blob/9b28a43d9f886653cc53daa312a48b8fddeea1b7/shared/localization/format.js#L146-L158)